### DISCUSSION TITLE
Flicking Docs translate.

### DISCUSSION THREAD

**Discussion by yongilkim01** [naver/egjs-flicking#764] [Q&A]
Currently, the page shows that the flicking API only supports version 4.10.4. Is it possible to translate version 4.10.4 into Korean?

**Comment by malangfox** [comment]
Hello @yongilkim01.

Our latest version 4.10.4 of the documentation has also been released in Korean and you can check it through the link below.
[https://naver.github.io/egjs-flicking/ko/docs/api/Flicking](https://naver.github.io/egjs-flicking/ko/docs/api/Flicking)
If you encounter any problems checking the documentation, please leave a comment.
